src/data/services/LmsApiService.js

```diff
@@ -30,20 +30,6 @@ class LmsApiService {
 
   static enterpriseCustomerInviteKeyUrl = `${LmsApiService.baseUrl}/enterprise/api/v1/enterprise-customer-invite-key/`;
 
-  static fetchCourseOutline(courseId) {
-    const options = {
-      course_id: courseId,
-      username: 'staff',
-      depth: 'all',
-      nav_depth: 3,
-      block_types_filter: 'course,chapter,sequential,vertical',
-    };
-    const queryParams = new URLSearchParams(options);
-
-    const outlineUrl = `${LmsApiService.baseUrl}/api/courses/v1/blocks/?${queryParams.toString()}`;
-    return LmsApiService.apiClient().get(outlineUrl);
-  }
```

Comment:
Unused function since 2018! Leftover from a POC on trying to bring courseware units/blocks into the MFE.